encryption/protocol.go

```diff
@@ -405,6 +405,8 @@ func (p *Protocol) GetPublicBundle(theirIdentityKey *ecdsa.PublicKey) (*Bundle,
 
 // ConfirmMessageProcessed confirms and deletes message keys for the given messages
 func (p *Protocol) ConfirmMessageProcessed(messageID []byte) error {
+	logger := p.logger.With(zap.String("site", "ConfirmMessageProcessed"))
+	logger.Debug("confirm message", zap.Binary("message-id", messageID))
```

Comment:
Shouldn't this read like this?

```suggestion
	logger.Debug("message confirmed", zap.Binary("message-id", messageID))
```